scripts/main/contextMenu.js

```diff
@@ -209,13 +220,16 @@ contextMenu.albumTitle = function (albumID, e) {
 };
 
 contextMenu.photo = function (photoID, e) {
-	// Notice for 'Move':
-	// fn must call basicContext.close() first,
-	// in order to keep the selection
+	let coverActive = photoID === album.json.cover_id;
 
 	let items = [
 		{ title: build.iconic("star") + lychee.locale["STAR"], fn: () => photo.setStar([photoID]) },
 		{ title: build.iconic("tag") + lychee.locale["TAGS"], fn: () => photo.editTags([photoID]) },
+		// for future work, use a list of all the ancestors.
```

Comment:
Is that actually needed? Selecting the cover one level at a time doesn't strike me as overly burdensome...

Reply:
Assume that you have an album without pictures but just sub albums...

Reply:
Actually, selecting cover of a subalbum is currently broken. I'm fixing it...